src/applications/mhv-secure-messaging/tests/components/Compose/ComposeForm.unit.spec.jsx

```diff
@@ -891,9 +892,13 @@ describe('Compose form component', () => {
     expect(screen.queryByTestId('send-button')).to.not.exist;
   });
 
-  it('displays alerts in Electronic Signature component if signature and checkbox is required', async () => {
+  it('displays alerts in Electronic Signature component if signature is required', async () => {
     const screen = renderWithStoreAndRouter(
-      <ComposeForm recipients={initialState.sm.recipients} />,
+      <ComposeForm
+        recipients={initialState.sm.recipients}
+        messageValid
+        isSignatureRequired
```

Comment:
I dont think `messageValid` and `isSignatureRequired` are valid props of this component?